### ISSUE TITLE
Allow wrapArguments --wrapconditions to split multiple conditions on new lines

### ISSUE THREAD

**Issue by davidelorenzitw** [nicklockwood/SwiftFormat#878]
It seems that this rule is not splitting multiple conditions, but only wrapping them (either before or after first one, depending on configuration) if they are already defined on multiple lines.
I guess this is the expected behaviour of the current implementation and not a bug, so probably this is more of a suggestion for a possible improvement.

With the source code:
```
guard let self = self, case let .success(value) = result, value.isEmpty else {
    // Whatever 
}
```
With the current implementation, it is left unmodified.
What I expected at first, and what I'm suggesting to have, is:
```
guard let self = self,
      case let .success(value) = result, 
      value.isEmpty else {
    // Whatever 
}
```

The same would probably apply to similar options, ie. `wraparguments` and `wrapparameters`.
Instead, `wrapcollections` seems already splitting automatically into multiple lines each item, if they appeared all on a single line. (Basically, this is the behaviour I expected for the above options too).

**Comment by mrunalabhatt** [comment]
I agree with david, same confusion i had while using this format option. 

Any update on this ?

**Comment by piro49** [comment]
Still no updates on this?

**Comment by dwirandytlvk** [comment]
any update?

**Comment by czottmann** [comment]
Yeah, I'd like to see this, too. (If possible.)
